src/mono/wasm/test-main.js

```diff
@@ -22,48 +22,45 @@ const originalConsole = {
     error: console.error
 };
 
-let isXUnitDoneCheck = false;
-let isXmlDoneCheck = false;
-
-function proxyMethod(prefix, func, asJson) {
+function proxyConsole(prefix, func, asJson) {
```

Comment:
nit: I think the earlier name `proxyMethod` made sense, because this is a function. Unlike `originalConsole`, which is an object with the console log methods.

Reply:
renamed to `proxyConsoleMethod`